### DISCUSSION TITLE
Integration with super-productivity

### DISCUSSION THREAD

**Discussion by a-b** [ActivityWatch/activitywatch#855] [Q&A]
I'm looking for any experience aggregating [super-productivity](https://github.com/johannesjo/super-productivity) data into Activity Watch.

[Related discussion](https://github.com/johannesjo/super-productivity/discussions/2527)

**Comment by BelKed** [comment]
ActivityWatch has an [API](https://docs.activitywatch.net/en/stable/api.html) that you can use to import data into the ActivityWatch database.
No idea how you can access Super Productivity data, though.
